doc/contributor/cexts.md

```diff
@@ -14,7 +14,7 @@ Clone Sulong as a sibling directory and build it:
 # In the truffleruby repo
 git clone https://github.com/graalvm/sulong.git ../sulong
 cd ../sulong
-mx build
+jt mx build
```

Comment:
It seems a bit magic to me that `jt build` would do two different things based on the current directory.